docs/concepts/securing-lokomotive-cluster.md

```diff
@@ -0,0 +1,53 @@
+# Securing Lokomotive clusters
+
+Lokomotive clusters are deployed with security as a top concern.
+
+## Cluster wide Pod Security Policy
+
+A [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) is a
+cluster-level resource that controls security sensitive aspects of the pod specification. The
+PodSecurityPolicy objects define a set of conditions that a pod must run with in order to be
+accepted into the system, as well as defaults for the related fields.
+
+Lokomotive clusters have PodSecurityPolicy (PSP) enabled by default. The cluster comes with two
+default PSPs for general purpose application usage:
+
+  * [restricted](https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-restricted.yaml)
```

Comment:
Can we make this a relative link instead of the full URL? Otherwise it'd go to the master version of the file.
```suggestion
  * [restricted](../../assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-restricted.yaml)
```